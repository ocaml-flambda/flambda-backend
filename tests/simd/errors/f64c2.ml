open Stdlib

type t = float64x2

external const2 : float -> float -> t = "" "caml_float64x2_const2"
        [@@noalloc] [@@unboxed] [@@builtin]

let _ = const2 0.0 (Sys.opaque_identity 1.0)
