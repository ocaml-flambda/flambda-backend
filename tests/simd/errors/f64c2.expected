File "f64c2.ml", line 1:
Error: Did not find constant float arguments for caml_float64x2_const2
